### PR TITLE
Awesome Books: working with classes PR

### PR DESCRIPTION
**Pull request by til2to**
## In this milestone, the following features were implemented:

- JavaScript classes instead of objects and arrays were used.
- class methods used for adding and removing books.
- Original functionality of the application was maintained.
- No use of library or framework.
- The application was styled to match the required wireframe

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  /* remove book */
+  remove(buttonId) {
+    this.allBooks = this.allBooks.filter((book, index) => index !== buttonId);
+    window.localStorage.setItem('allBooks', JSON.stringify(this.allBooks));
+    window.location.reload();
+  }
+}
+/* eslint-disable */
+const freshBook = new BookList();
```
Comment (GzimAsani) [inline script.js]: - [ ] The app is having a problem when the title and author are duplicated. It is adding the component more than 1 time. The same thing is happening when removing the duplication. Kindly check your logic and try to fix this issue. Check the video for illustration:
https://www.loom.com/share/d8c3b08e2b044f878feb9cc2ccae9864

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @til2to and @aubinleyoung                            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job adding classes and constructor :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Whoistolu) [reply script.js]: - [ ] Well resolved @til2to and @aubinleyoung 
Screenshot 2023-01-11 at 18 11 19

**Review by Whoistolu — APPROVED** [review]
### Hi @til2to and @aubinleyoung 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**